### ISSUE TITLE
Connects only first time

### ISSUE THREAD

**Issue by Tomasselli** [ipatix/libvirt-usb-hotplug#1]
It works fine, but only first time i plug in an usb device. Second time, the device mounts again on the host machine. I have Zorin as Host OS and Windows 10 as Guest running in KVM. After rebooting the Guest OS it works again one time.

**Comment by ipatix** [comment]
Yes, that is unfortunately a limitation. To solve this one would probably need a daemon or something that wait's for VM changes since the devices cannot be attached to the VM if it's still turned off (and once the udev event trigger is finished, no record of these devices is kept).

I originally wrote this during a time where I did a lot of tinkering with libvirt, but I currently do not actively work on this hence why I can't promise anything on when exactly I'll implement this. Though, I'll keep the issue open for the meantime.

EDIT:
Perhaps I misread your problem. The issue I'm already aware of is that you can't plug in a device to a VM if it's turned off. It's always required to replug the device after the VM is turned on, but usually there should not be a problem if the VM is already running.

**Comment by Tomasselli** [comment]
Thanks a lot for your answer. When the VM ist running and i plug in a usb device, it works. 
The problem actually is: Do i unplug and replug the device, it doesn't work/mount again. I have to restart the VM to get it work again.
